backends/qualcomm/quantizer/observers/per_channel_param_observer.py

```diff
@@ -0,0 +1,104 @@
+import torch
+from torch.ao.quantization.observer import UniformQuantizationObserverBase
+
+
+# TODO move to torch/ao/quantization/observer.py.
+class PerChannelParamObserver(UniformQuantizationObserverBase):
```

Comment:
Can you add some comments here as to why chose the min/max ranges in the way being done here?

Reply:
Hi @kimishpatel, just add the comment. If it's not clear enought please feel free to let me know. Thanks